app/controllers/admin/categories_controller.rb

```diff
@@ -25,12 +25,17 @@ def destroy
 
   def new_or_edit
     @categories = Category.find(:all)
-    @category = Category.find(params[:id])
+    @category = case params[:id]
```

Comment:
This is a bit of an odd use of a case statement since it is only two conditions, might recommend an `if/else` instead